css_def/def.go

```diff
@@ -0,0 +1,48 @@
+package css_def
```

Comment:
It seems that this file (`css_def/def.go`) is no longer needed (see PR #1906 in sourcegraph)